Update resources.md

ContentBrewmaster opened the pull request:
Based on recent research about where employees look for information related to discrimination, adding a link to CivicActions' code of conduct, with a a description captured word-for-word from the the Code of Conduct page.

Also removed bullets to address formatting issue and changed title to sentence case to align with our writing style.

<!-- readthedocs-preview civicactions-handbook start -->
----
:books: Documentation preview :books:: https://civicactions-handbook--1032.org.readthedocs.build/en/1032/

<!-- readthedocs-preview civicactions-handbook end -->

ContentBrewmaster committed:
Update resources.md

Based on recent research about where employees look for information related to discrimination, adding a link to CivicActions' code of conduct, with a a description captured word-for-word from the the Code of Conduct page.

Also removed bullets to address formatting issue and changed title to sentence case to align with our writing style.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra commented on about-civicactions/diversity-equity-inclusion/resources.md:
```diff
@@ -1,17 +1,28 @@
+
+What is equity, and how can our organization support a more equitable tech industry? Check out the presentation introducing the equity workshop at the 2019 retreat.
+
+\*\*[Code of conduct](https://guidebook.civicactions.com/en/latest/company-policies/code-of-conduct/)
```
@ContentBrewmaster can this link be relative instead of absolute i.e. point to the markdown file for code of conduct? Also it missing the `**` at the end to make it bold.

dmundra reviewed (changes requested):
In the files tab you can also see the complaint about using emphasis. Can that be changed to heading as suggested or an alternate approach?

ContentBrewmaster committed:
Update resources.md

Removed emphasis from links and redid link for Code of Conduct as relative link.

dmundra reviewed (approved):
The read the docs preview is broken but the changes looks good to me to continue and then make changes after it is deployed as needed. Approved!

dmundra committed:
Merge branch 'master' into ContentBrewmaster-patch-2
